src/games/strategy/engine/xml/TestGameLoader.java

```diff
@@ -35,7 +35,7 @@
    * The game is about to start.
    */
   @Override
-  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless) {}
+  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless, Runnable exceptionAction) {}
```

Comment:
This interface could use some reworking, the boolean parameter for 'headless' is a hack.

Really we have these two APIs bundled into just one:
```
public void startGame(final IGame game, final Set<IGamePlayer> players);
public void startHeadlessGame(final IGame game, final Set<IGamePlayer> players)
```

Likely that split is a first step. Then I would aggregate the stuff in common, and restructure to use composition to execute the common parts.

For now though I think it would be better to return a status result, instead of adding a parameter that is sometimes used, sometimes not.